LoopFollow/Remote/Loop/LoopOverrideView.swift

```diff
@@ -0,0 +1,222 @@
+//
+//  LoopOverrideView.swift
+//  LoopFollow
+//
+//  Created by Jonas Björkert on 2025-01-14.
+//  Copyright © 2025 Jon Fawcett. All rights reserved.
+//
+
+import SwiftUI
+import HealthKit
+
+struct LoopOverrideView: View {
+    @Environment(\.presentationMode) private var presentationMode
+
+    @ObservedObject var device = ObservableUserDefaults.shared.device
+    @ObservedObject var overrideNote = Observable.shared.override
+    @ObservedObject var nsAdmin = ObservableUserDefaults.shared.nsWriteAuth
+
+    @StateObject private var viewModel = LoopOverrideViewModel()
+
+    @State private var showAlert: Bool = false
+    @State private var alertType: AlertType? = nil
+    @State private var alertMessage: String? = nil
+    @State private var isLoading: Bool = false
+    @State private var statusMessage: String? = nil
+
+    @State private var selectedOverride: ProfileManager.LoopOverride? = nil
+    @State private var showConfirmation: Bool = false
+
+    @FocusState private var noteFieldIsFocused: Bool
+
+    private let pushNotificationManager = PushNotificationManager()
+    private var profileManager = ProfileManager.shared
+
+    enum AlertType {
+        case confirmActivation
+        case confirmCancellation
+        case statusSuccess
+        case statusFailure
+        case validation
+    }
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                if device.value != "Loop" {
+                    ErrorMessageView(
+                        message: "Remote commands are currently only available for Loop."
+                    )
+                } else if !nsAdmin.value {
+                    ErrorMessageView(
+                        message: "Please update your token to include the 'admin' role in order to do remote commands."
```

Comment:
add word Loop because this is specific to Loop.
find the equivalent for Trio and add the word Trio (careportal and readable is sufficient)